lib/oauth2.js

```diff
@@ -145,16 +144,23 @@ exports.OAuth2.prototype._executeRequest= function( http_library, options, post_
 
   var request = http_library.request(options);
   request.on('response', function (response) {
-    response.on("data", function (chunk) {
+    let responseHandler = response
```

Comment:
Not important, but you forgot something: `;` 😛

Reply:
Right. Also the `let` here looks inconsistent, as all variables are declared with `var` in this file.